addons/binding/org.openhab.binding.allplay/src/main/java/org/openhab/binding/allplay/handler/AllPlayHandler.java

```diff
@@ -410,9 +410,9 @@ private void updateCurrentItemState(PlaylistItem currentItem) {
         try {
             updateState(CURRENT_USER_DATA, new StringType(String.valueOf(currentItem.getUserData())));
         } catch (SpeakerException e) {
-            logger.warn("Unable to update current user data: {}", e.getMessage(), e);
+            logger.warn("Unable to update current user data: ", e);
```

Comment:
Then we should exclude these pattern : https://github.com/KengoTODA/findbugs-slf4j#slf4j_manually_provided_message